romancal/stpipe/core.py

```diff
@@ -69,6 +69,25 @@ def _get_crds_parameters(cls, dataset):
             ]
         return crds_parameters, crds_observatory
 
+    @staticmethod
+    def get_stpipe_loggers():
+        """
+        Get names of loggers stpipe will configure
+
+        List the names of all loggers that might emit
+        messages useful to display or record to the datamodel.
+
+        Since messages are often propagated up the logging
+        heirarchy the names here can be the names of packages
+        that might emit useful log messages.
+
+        Returns
+        -------
+        tuple of str
+            Tuple of log names for stpipe to configure
+        """
+        return ("romancal", "stcal", "roman_datamodels", "stpipe", "tweakwcs")
```

Comment:
@schlafly @ddavis-stsci What packages might emit log messages that we want to display to users and log to datamodels during `strun ...` calls, `Step.call` executions, etc?

Reply:
Thanks. I will bring this out of draft and run the regtests once the corresponding stpipe PR is merged.